test/witan/send/vis_test.clj

```diff
@@ -1,7 +0,0 @@
-(ns witan.send.vis-test
```

Comment:
could do with some tests...

Reply:
Yeah, some of the data transformation stuff could use some in particular.